### PR TITLE
feat: Add a workflow for greetings when users first interactive with repo

### PR DESCRIPTION
**Pull request by cychiang**
#### Summary
This PR is based on the issue #3161 

#### Use the following steps to ensure your PR is ready to be reviewed
- [x] Read the [contribution guidelines](../blob/main/CONTRIBUTING.md) 🧑‍🎓
- [x] Test locally on your own infrastructure
- [x] Run `go fmt` to format your code 🖊
- [x] Lint your changes via `golangci-lint run` 🚨 (install golangci-lint [here](https://golangci-lint.run/usage/install/#local-installation))
- [x] Update or add tests 🧪
- [ ] Ensure the status checks below are successful ✅

### CODE REVIEW

File: .github/workflows/greetings.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+jobs:
+  greeting:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/first-interaction@v1
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          issue-message: ""
+          pr-message: ""
```
Comment (erezrokah) [inline .github/workflows/greetings.yml:13]:
```suggestion
          issue-message: "Thanks for reporting this issue 👍 You can reach us via [Discord](https://www.cloudquery.io/discord) too. If you enjoy using this project, please consider staring it for support"
```

**Review by erezrokah — CHANGES_REQUESTED** [review]
Thanks @cychiang lets start only with an issue comment.

Please let me know what you think

File: .github/workflows/greetings.yml (hunk 1 above)
Comment (cychiang) [reply .github/workflows/greetings.yml]: Done 👍

**Review by erezrokah — APPROVED** [review]
This looks good to me, but I'll wait for others to approve too, cc @cloudquery/cloudquery-opensource

**Review by yevgenypats — APPROVED** [review]

**Pull request merged**